chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -257,7 +257,7 @@
           end
 
         when os =~ /^(open)?suse/
-          append << "install=#{install_url} autoyast=#{node_url}/autoyast.xml"
+          append << "install=#{install_url} autoyast=#{node_url}/autoyast-upgrade.xml autoupgrade=1"
```

Comment:
Shouldn't we check somehow if this upgrade is needed otherwise we could use standard `autoyast.xml` template